src/sass/app_2/_common/components/theme-select.scss

```diff
@@ -0,0 +1,61 @@
+.theme-select-settings {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    width: 100%;
+
+    &__title {
+        font-weight: bold;
+        font-size: 14px;
+        color: $COLOR_LIGHT_BLACK_1;
+        margin-bottom: 17px;
+    }
+    &__content {
+        display: flex;
+        justify-content: space-between;
+        align-items: flex-start;
+        width: 100%;
+    }
+    &__option {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        width: 234px;
+        height: 207px;
+
+        &__title {
+            margin-top: 9px;
+            font-size: 14px;
+            line-height: 1.43;
+        }
+        &__icon {
+            width: 234px;
+            height: 178px;
+            cursor: pointer;
+            border-radius: 4px;
+            border: 1px solid $COLOR_LIGHT_GRAY_1;
+        }
+        &__icon--active, &__icon:hover {
+            box-shadow: 0 0 0 2px $COLOR_LIGHT_GRAY_2;
+        }
+    }
+    .theme--dark & {
+        &__title {
```

Comment:
should this be moved to `dark.scss` instead?

Reply:
could you use `themify` for the dark color `themed('text_primary_color')`